tests/test_flask_server.py

```diff
@@ -0,0 +1,44 @@
+from blockchain.network import server
+from blockchain.config import CONFIG
+from blockchain.full_client import FullClient
+
+
+def setup_module(module):
+    server.app.config['TESTING'] = True
+    module.client = server.app.test_client()
+    server.full_client = FullClient()
+    module.routes_to_test = CONFIG.ROUTES
+
+    # ROUTES = {
```

Comment:
Should this give an overview of the present routes (comments also get stale :/) or is this a leftover from working on the implementation?

Reply:
:+1: that is a leftover from the implementation.